docs/aerodromes/classc/Amberley.md

```diff
@@ -72,17 +110,42 @@ The Circuit Area Airspace is allocated to be within 5nm of the YAMB ARP from `SF
 | 04     | Left |
 | 22     | Left |
 
+### Civil Operations
+Civil and AAFC conduct flying operations within the South Western Training Area (SWTA). The SWTA is divided in half by the Rosewood-Mt Walker-Aratula Road, SWTA Alpha (north) and SWTA Bravo (south). Once established in the SWTA, aircraft will remain OCTA. Pilots may request to enter the CTA and climb above `A025` when operationally required.
+
+Departures:
+
+| **Duty Runway** | **Routing** | **Altitude** |
+|-----------------|-----------|--------------|
+| 15              | MTWK      | `A025`         |
+| 33              | CLVT      | `A025`        |
+
+Arrivals:
+
+| **Duty Runway** | **Routing** | **Altitude**         |
+|-----------------|-----------|----------------------|
+| 15              | CLVT      | `A025` or `A015`        |
+| 33              | MTWK      | `A025` or `A015`       |
+
+!!! phraseology 
+    **AMB ADC**: "ASTR203, cleared SWTA Alpha via MTWK, maintain 2500, squawk 7301"  
+    **ASTR203**: "Cleared SWTA Alpha via MTWK, maintain 2500, squawk 7301, ASTR203"    
```

Comment:
This might be better on the TCU page, but a section highlighting the training area would be useful for ADC so they know how to clear an aircraft there

Reply:
So just confirmed, ADC page should include the overview, and phraseology example. While the TCU page includes routing and phraseology examples?

Reply:
Outbound routing absolutely, good to have for ADC so they can issue appropriate clearances. Inbound routing is good for situational awareness but maybe a note to indicate that TCU will take care of it (I'm assuming this is how it works?). The note about aircraft climbing into CTA above A025 is irrelevant to ADC who only owns to A015, but TCU needs to be aware if this is a common procedure. Also need TCU to be aware of routing so they can issue appropriate inbound clearances (again, assuming this is how its managed).

Reply:
Sounds good, I will add routing into the TCU and move the climb into CTA note to TCU. Plus add the note for arrivals routing being handled by the TCU into ADC.

Reply:
Added the routing, climb into CTA, aircraft entering / leaving SWTA procedure.

Reply:
Thanks mate